### ISSUE TITLE
feat: Revision Update

### ISSUE THREAD

**Issue by dberuben** [seacrew/helm-compose#18]
Hi,

The revision is updated even when there are no updates in the helm compose file.
It's possible to have revision updated only when there is some things new in the config?

```
(☸|lspay-trial-eu-west-1:default)➜  helm-compose helm compose up
helloworld5 |		Release "helloworld5" has been upgraded. Happy Helming!
helloworld5 |		NAME: helloworld5
helloworld5 |		LAST DEPLOYED: Thu Aug 31 15:51:04 2023
helloworld5 |		NAMESPACE: default
helloworld5 |		STATUS: deployed
helloworld5 |		REVISION: 7
helloworld5 |		TEST SUITE: None
helloworld4 |		Release "helloworld4" has been upgraded. Happy Helming!
helloworld4 |		NAME: helloworld4
helloworld4 |		LAST DEPLOYED: Thu Aug 31 15:51:04 2023
helloworld4 |		NAMESPACE: default
helloworld4 |		STATUS: deployed
helloworld4 |		REVISION: 7
helloworld4 |		TEST SUITE: None
helloworld7 |		Release "helloworld7" has been upgraded. Happy Helming!
helloworld7 |		NAME: helloworld7
helloworld7 |		LAST DEPLOYED: Thu Aug 31 15:51:04 2023
helloworld7 |		NAMESPACE: default
helloworld7 |		STATUS: deployed
helloworld7 |		REVISION: 7
helloworld7 |		TEST SUITE: None
helloworld6 |		Release "helloworld6" has been upgraded. Happy Helming!
helloworld6 |		NAME: helloworld6
helloworld6 |		LAST DEPLOYED: Thu Aug 31 15:51:04 2023
helloworld6 |		NAMESPACE: default
helloworld6 |		STATUS: deployed
helloworld6 |		REVISION: 7
helloworld6 |		TEST SUITE: None
helloworld3 |		Release "helloworld3" has been upgraded. Happy Helming!
helloworld3 |		NAME: helloworld3
helloworld3 |		LAST DEPLOYED: Thu Aug 31 15:51:04 2023
helloworld3 |		NAMESPACE: default
helloworld3 |		STATUS: deployed
helloworld3 |		REVISION: 7
helloworld3 |		TEST SUITE: None
helloworld1 |		Release "helloworld1" has been upgraded. Happy Helming!
helloworld1 |		NAME: helloworld1
helloworld1 |		LAST DEPLOYED: Thu Aug 31 15:51:04 2023
helloworld1 |		NAMESPACE: default
helloworld1 |		STATUS: deployed
helloworld1 |		REVISION: 7
helloworld1 |		TEST SUITE: None
helloworld2 |		Release "helloworld2" has been upgraded. Happy Helming!
helloworld2 |		NAME: helloworld2
helloworld2 |		LAST DEPLOYED: Thu Aug 31 15:51:04 2023
helloworld2 |		NAMESPACE: default
helloworld2 |		STATUS: deployed
helloworld2 |		REVISION: 7
helloworld2 |		TEST SUITE: None
(☸|lspay-trial-eu-west-1:default)➜  helm-compose helm ls
NAME       	NAMESPACE	REVISION	UPDATED                              	STATUS  	CHART            	APP VERSION
helloworld1	default  	7       	2023-08-31 15:51:04.050979 +0200 CEST	deployed	hello-world-0.6.0
helloworld2	default  	7       	2023-08-31 15:51:04.103239 +0200 CEST	deployed	hello-world-0.6.0
helloworld3	default  	7       	2023-08-31 15:51:04.141622 +0200 CEST	deployed	hello-world-0.6.0
helloworld4	default  	7       	2023-08-31 15:51:04.06492 +0200 CEST 	deployed	hello-world-0.6.0
helloworld5	default  	7       	2023-08-31 15:51:04.01752 +0200 CEST 	deployed	hello-world-0.6.0
helloworld6	default  	7       	2023-08-31 15:51:04.101631 +0200 CEST	deployed	hello-world-0.6.0
helloworld7	default  	7       	2023-08-31 15:51:04.143614 +0200 CEST	deployed	hello-world-0.6.0
(☸|lspay-trial-eu-west-1:default)➜  helm-compose helm compose list
| Date             | Revision |
| ---------------- | -------- |
| 2023-08-31 15:29 |        4 |
| 2023-08-31 15:30 |        5 |
| 2023-08-31 15:36 |        6 |
| 2023-08-31 15:45 |        7 |
| 2023-08-31 15:47 |        8 |
| 2023-08-31 15:48 |        9 |
| 2023-08-31 15:48 |       10 |
| 2023-08-31 15:49 |       11 |
| 2023-08-31 15:49 |       12 |
| 2023-08-31 15:51 |       13 |
```

```
apiVersion: 1.0

storage:
  name: mycompose
  type: local # default
  path: .hcstate # default

releases:
  helloworld1:
    chart: sikalabs/hello-world 
    values:
      replicas: 2
  helloworld2:
    chart: sikalabs/hello-world 
    values:
      replicas: 2
  helloworld3:
    chart: sikalabs/hello-world 
    values:
      TEXT:
        COUCOUL3: DANIELTEST
      replicas: 1
  helloworld4:
    chart: sikalabs/hello-world 
    values:
      replicas: 2
  helloworld5:
    chart: sikalabs/hello-world 
    values:
      TEXT:
        COUCOUL5: DANIELTEST
      replicas: 2
  helloworld6:
    chart: sikalabs/hello-world 
    values:
      replicas: 2
  helloworld7:
    chart: sikalabs/hello-world 
    values:
      replicas: 2
repositories:
  sikalabs: https://helm.sikalabs.io

```

**Comment by tuunit** [comment]
Hi @dberuben,

first of all thanks for all the testing 😊 I'll check it out later!

**Comment by tuunit** [comment]
Fixed with b576fc1 will be released shortly

**Comment by dberuben** [comment]
You r0x Thank you @tuunit
